channel/storage.go

```diff
@@ -0,0 +1,296 @@
+package channel
+
+import (
+	"database/sql"
+	"fmt"
+	"sort"
+
+	"github.com/Hucaru/Valhalla/common"
+)
+
+// Storage capacity bounds
+const (
+	storageMinSlots byte = 20
+	storageMaxSlots byte = 255
+)
+
+type storage struct {
+	accountID int32
```

Comment:
I would remove the account ID from here since this struct is a member of the player struct, you can then pass the accountID to the save/load functions and save yourself the accountID value check.